orb_models/dataset/data_loaders.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
Can I suggest moving these functions to the `finetune.py` script? We want to minimize things that we will support forever, and this is really just boilerplate for the dataset creation